snap/snapcraft.yaml

```diff
@@ -156,6 +158,21 @@ apps:
     daemon: oneshot
     environment:
       INIT_ARG: "--vaultInterval=10"
+      SecretService_Server: localhost
+      SecretService_ServerName: localhost
+      SecretService_CaFilePath: $SNAP_DATA/pki/ca/ca.pem
+      SecretService_CertFilePath: $SNAP_DATA/pki/EdgeXFoundryCA/localhost.pem
+      SecretService_KeyFilePath: $SNAP_DATA/pki/EdgeXFoundryCA/localhost.priv.key
+      SecretService_TokenFolderPath: $SNAP_DATA/config/security-secrets-setup/res
+      SecretService_TokenProvider: $SNAP/bin/security-file-token-provider
+      SecretService_TokenProviderArgs: "-confdir, $SNAP_DATA/config/security-file-token-provider/res"
+      SecretService_TokenProviderAdminTokenPath: $SNAP_DATA/secrets/tokenprovider/secrets-token.json
+
+      # environment for security-file-token-provider, exec'd by secretstore-setup
+      TokenFileProvider_PrivilegedTokenPath: $SNAP_DATA/secrets/tokenprovider/secrets-token.json
+      TokenFileProvider_ConfigFile: $SNAP_DATA/config/security-file-token-provider/res/token-config.json
+      TokenFileProvider_OutputDir: $SNAP_DATA/secrets
+
```

Comment:
I unfortunately don't have the bandwidth to manually audit that all of these are the right keys/values, but they look like the right thing to me fwiw